commit 42e89e3fdb9cd83a1c6d6ec826c79c817f118aef
Author: mb-karo <mb@karo-electronics.de>
Date:   Mon Jan 20 14:48:02 2020 +0100

    Modify TX8M-1610 and TX8M-ND00 defconfig to autostart also QT5 applications and weston desktop.

diff --git a/arch/arm64/configs/tx8m-1610_defconfig b/arch/arm64/configs/tx8m-1610_defconfig
index 976a1c8..2472a2b 100644
--- a/arch/arm64/configs/tx8m-1610_defconfig
+++ b/arch/arm64/configs/tx8m-1610_defconfig
@@ -267,11 +267,12 @@ CONFIG_MEDIA_CAMERA_SUPPORT=y
 CONFIG_MEDIA_CONTROLLER=y
 CONFIG_VIDEO_ADV_DEBUG=y
 CONFIG_V4L_PLATFORM_DRIVERS=y
+CONFIG_IMX_DPU_CORE=y
+CONFIG_IMX_DCSS_CORE=y
 CONFIG_IMX_LCDIF_CORE=y
 CONFIG_DRM=y
-CONFIG_DRM_VIVANTE=y
-CONFIG_DRM_PANEL_SIMPLE=y
 CONFIG_DRM_IMX=y
+CONFIG_DRM_IMX_PARALLEL_DISPLAY=y
 CONFIG_DRM_IMX_SEC_DSIM=y
 CONFIG_FB_IMX64=y
 CONFIG_FB_IMX64_DEBUG=y
@@ -353,8 +354,6 @@ CONFIG_MMC_SDHCI_PLTFM=y
 CONFIG_MMC_SDHCI_ESDHC_IMX=y
 CONFIG_MMC_SPI=y
 CONFIG_MXC_VPU_MALONE_DEBUG=y
-CONFIG_MXC_VPU_DECODER_DEBUG=y
-CONFIG_MXC_VPU_ENCODER_DEBUG=y
 CONFIG_NEW_LEDS=y
 CONFIG_LEDS_CLASS=y
 CONFIG_LEDS_GPIO=y
@@ -372,6 +371,12 @@ CONFIG_DMADEVICES_DEBUG=y
 CONFIG_DMADEVICES_VDEBUG=y
 CONFIG_FSL_EDMA_V3=y
 CONFIG_IMX_SDMA=y
+CONFIG_STAGING=y
+CONFIG_ION=y
+CONFIG_ION_SYSTEM_HEAP=y
+CONFIG_ION_CARVEOUT_HEAP=y
+CONFIG_ION_CHUNK_HEAP=y
+CONFIG_ION_CMA_HEAP=y
 # CONFIG_COMMON_CLK_XGENE is not set
 # CONFIG_HISILICON_ERRATUM_161010101 is not set
 CONFIG_ARM_SMMU=y
@@ -430,9 +435,9 @@ CONFIG_CRYPTO_RSA=y
 # CONFIG_CRYPTO_MANAGER_DISABLE_TESTS is not set
 CONFIG_CRYPTO_AUTHENC=y
 # CONFIG_CRYPTO_ECHAINIV is not set
-CONFIG_CRYPTO_DES=y
+CONFIG_CRYPTO_ANSI_CPRNG=y
+CONFIG_CRYPTO_USER_API_RNG=y
 CONFIG_CRYPTO_DEV_FSL_CAAM=y
-# CONFIG_CRYPTO_DEV_FSL_CAAM_JR is not set
 # CONFIG_CRYPTO_DEV_VIRTIO is not set
 CONFIG_ARM64_CRYPTO=y
 CONFIG_CRYPTO_SHA1_ARM64_CE=y
diff --git a/arch/arm64/configs/tx8m-nd00_defconfig b/arch/arm64/configs/tx8m-nd00_defconfig
index 1d39400..dfa20ac 100644
--- a/arch/arm64/configs/tx8m-nd00_defconfig
+++ b/arch/arm64/configs/tx8m-nd00_defconfig
@@ -268,11 +268,12 @@ CONFIG_MEDIA_CAMERA_SUPPORT=y
 CONFIG_MEDIA_CONTROLLER=y
 CONFIG_VIDEO_ADV_DEBUG=y
 CONFIG_V4L_PLATFORM_DRIVERS=y
+CONFIG_IMX_DPU_CORE=y
+CONFIG_IMX_DCSS_CORE=y
 CONFIG_IMX_LCDIF_CORE=y
 CONFIG_DRM=y
-CONFIG_DRM_VIVANTE=y
-CONFIG_DRM_PANEL_SIMPLE=y
 CONFIG_DRM_IMX=y
+CONFIG_DRM_IMX_PARALLEL_DISPLAY=y
 CONFIG_DRM_IMX_SEC_DSIM=y
 CONFIG_FB_IMX64=y
 CONFIG_FB_IMX64_DEBUG=y
@@ -353,6 +354,7 @@ CONFIG_MMC_SDHCI=y
 CONFIG_MMC_SDHCI_PLTFM=y
 CONFIG_MMC_SDHCI_ESDHC_IMX=y
 CONFIG_MMC_SPI=y
+CONFIG_MXC_VPU_MALONE_DEBUG=y
 CONFIG_NEW_LEDS=y
 CONFIG_LEDS_CLASS=y
 CONFIG_LEDS_GPIO=y
@@ -370,6 +372,12 @@ CONFIG_DMADEVICES_DEBUG=y
 CONFIG_DMADEVICES_VDEBUG=y
 CONFIG_FSL_EDMA_V3=y
 CONFIG_IMX_SDMA=y
+CONFIG_STAGING=y
+CONFIG_ION=y
+CONFIG_ION_SYSTEM_HEAP=y
+CONFIG_ION_CARVEOUT_HEAP=y
+CONFIG_ION_CHUNK_HEAP=y
+CONFIG_ION_CMA_HEAP=y
 # CONFIG_COMMON_CLK_XGENE is not set
 # CONFIG_HISILICON_ERRATUM_161010101 is not set
 CONFIG_ARM_SMMU=y
@@ -428,9 +436,9 @@ CONFIG_CRYPTO_RSA=y
 # CONFIG_CRYPTO_MANAGER_DISABLE_TESTS is not set
 CONFIG_CRYPTO_AUTHENC=y
 # CONFIG_CRYPTO_ECHAINIV is not set
-CONFIG_CRYPTO_DES=y
+CONFIG_CRYPTO_ANSI_CPRNG=y
+CONFIG_CRYPTO_USER_API_RNG=y
 CONFIG_CRYPTO_DEV_FSL_CAAM=y
-# CONFIG_CRYPTO_DEV_FSL_CAAM_JR is not set
 # CONFIG_CRYPTO_DEV_VIRTIO is not set
 CONFIG_ARM64_CRYPTO=y
 CONFIG_CRYPTO_SHA1_ARM64_CE=y
